language-support/ts/daml-react/createLedgerContext.ts

```diff
@@ -24,6 +25,7 @@ export type LedgerProps = {
   token: string;
   httpBaseUrl?: string;
   wsBaseUrl?: string;
+  user: User;
```

Comment:
Does this mean I can no longer use `@daml/react` against a ledger without user management? If so, I think this is unfortunately not an option for 2.0

Reply:
You don't require user management from the ledger, but you would have to fill this value from the credentials in some way. The other option would be to make the value optional.

Reply:
Yes, this is fine. I'll change it.

Reply:
I think I’m confused it’s optional in `DamlLedgerState` but required in `LedgerProps`. What am I missing?

Reply:
Nothing, it should be optional in both, I missed it in the latter.